modules/security/pom.xml

```diff
@@ -52,5 +52,10 @@
             <artifactId>wiremock-standalone</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
can we remove this dependency if nimbus is not used anymore?

Reply:
Removed